diff --git a/client/composables/gitlog.ts b/client/composables/gitlog.ts
index 5717d0b98dd2da6270241ffadfc2b68b83356bea..3c9e5d5259a7d52031b305bd58eaaf31872e84c9 100644
--- a/client/composables/gitlog.ts
+++ b/client/composables/gitlog.ts
@@ -24,7 +24,7 @@ export function useGitLog(): Ref<GitLog> {
   const path = frontmatter.value.git_log?.path
 
   if (isPrebuilt && path) {
-    fetch('/git-log.json')
+    fetch('/docs/git-log.json')
       .then(res => res.json())
       .then((data: GitLogFileEntry) => {
         if (data[path])
